### ISSUE TITLE
need api key for the img if we are going to show this in the details for the final choice

### ISSUE THREAD

**Issue by abreaw** [TheCoolKidsUNC/DateNightApp#98]
Image URL returned is accessed w/ an API per the documentation --> http://developer.tmsapi.com/Sample_Code

var movieData = '<div class="tile"><img src="http://developer.tmsimg.com/' + movie.preferredImage.uri + '?api_key='+apikey+'"><br/>';

the current API Key is returning access denied error -->
GET http://developer.tmsimg.com/assets/p13019916_p_v5_aa.jpg?api_key=x23n96frwcyfg7jnss9h8ax4 403 (Forbidden)

**Comment by abreaw** [comment]
Actually after further look the picture might be additional plan required as shown in the get started page --> http://developer.tmsapi.com/Getting_Started
Sample Plan required for images it looks like

**Comment by Jnifong** [comment]
Yeah I just put a date in there for MVP. If you want I think adding a global variable for date could be done and then I can just change the API call to look at that number it passes in? Thoughts
